Cannot (de-)serialize rational64 with serde

Hello,

I am trying to (de-)serialize a `uom::si::rational64::Frequency` with serde but I cannot get it working.

Cargo.toml
```TOML
[package]
name = "uom-serde"
version = "0.1.0"
edition = "2018"

[dependencies]
uom = { version = "0.31.1", default-features = false, features = [ "autoconvert", "rational64", "si", "std", "use_serde"]}
serde = { version = "1.0.130", features = ["derive"] }
```
lib.rs
```rust
use serde::{Deserialize, Serialize};

#[derive(Serialize, Deserialize)]
pub struct Dummy {
    pub frequency: uom::si::rational64::Frequency,
}
```

Error:
```console
cargo build
   Compiling uom v0.31.1
   Compiling uom-serde v0.1.0 
error[E0277]: the trait bound `Ratio<i64>: Serialize` is not satisfied
 --> src\lib.rs:5:5
  |
5 |     pub frequency: uom::si::rational64::Frequency,
  |     ^^^ the trait `Serialize` is not implemented for `Ratio<i64>`
  |
  = note: required because of the requirements on the impl of `Serialize` for `Quantity<dyn Dimension<M = Z0, J = Z0, N = Z0, I = Z0, T = NInt<UInt<UTerm, B1>>, L = Z0, Th = Z0, Kind = (dyn Kind + 'static)>, dyn uom::si::Units<Ratio<i64>, length = uom::si::length::meter, 
electric_current = uom::si::electric_current::ampere, luminous_intensity = uom::si::luminous_intensity::candela, thermodynamic_temperature = uom::si::thermodynamic_temperature::kelvin, mass = uom::si::mass::kilogram, time = uom::si::time::second, amount_of_substance = uom::si::amount_of_substance::mole>, Ratio<i64>>`
  = note: required by `_::_serde::ser::SerializeStruct::serialize_field`

error[E0277]: the trait bound `Ratio<i64>: Deserialize<'_>` is not satisfied
 --> src\lib.rs:5:5
  |
5 |     pub frequency: uom::si::rational64::Frequency,
  |     ^^^ the trait `Deserialize<'_>` is not implemented for `Ratio<i64>`
  |
  = note: required because of the requirements on the impl of `Deserialize<'_>` for `Quantity<(dyn Dimension<M = Z0, J = Z0, N = Z0, I = Z0, T = NInt<UInt<UTerm, B1>>, L = Z0, Th = Z0, Kind = (dyn Kind + 'static)> + 'static), (dyn uom::si::Units<Ratio<i64>, length = uom::si::length::meter, electric_current = uom::si::electric_current::ampere, luminous_intensity = uom::si::luminous_intensity::candela, thermodynamic_temperature = uom::si::thermodynamic_temperature::kelvin, mass = uom::si::mass::kilogram, time = uom::si::time::second, amount_of_substance = uom::si::amount_of_substance::mole> + 'static), Ratio<i64>>`
  = note: required by `next_element`

error[E0277]: the trait bound `Ratio<i64>: Deserialize<'_>` is not satisfied
 --> src\lib.rs:5:5
  |
5 |     pub frequency: uom::si::rational64::Frequency,
  |     ^^^ the trait `Deserialize<'_>` is not implemented for `Ratio<i64>`
  |
  = note: required because of the requirements on the impl of `Deserialize<'_>` for `Quantity<(dyn Dimension<M = Z0, J = Z0, N = Z0, I = Z0, T = NInt<UInt<UTerm, B1>>, L = Z0, Th = Z0, Kind = (dyn Kind + 'static)> + 'static), (dyn uom::si::Units<Ratio<i64>, length = uom::si::length::meter, electric_current = uom::si::electric_current::ampere, luminous_intensity = uom::si::luminous_intensity::candela, thermodynamic_temperature = uom::si::thermodynamic_temperature::kelvin, mass = uom::si::mass::kilogram, time = uom::si::time::second, amount_of_substance = uom::si::amount_of_substance::mole> + 'static), Ratio<i64>>`
  = note: required by `next_value`

error: aborting due to 3 previous errors
```

As far as I can see from the Cargo.lock file, the `serde` feature of `num-rational` is not activated.

--- a/README.md
+++ b/README.md
@@ -110,6 +110,15 @@ uom = {
  * `use_serde` -- Feature to enable support for serialization and deserialization of quantities
    with the [Serde][serde] crate. Disabled by default.
 
+   [Serde][serde] support for the `big*` and `rational*` underlying storage types requires manually
+   enabling the `serde` feature for the `num-rational` and `num-bigint` crates. To do so, you can
+   add one or both of the following lines to your `Cargo.toml`:
+
+   ```toml
+   num-rational = { version = "*", features = ["serde"] }
+   num-bigint = { version = "*", features = ["serde"] }
+   ```
+
 [si]: https://jcgm.bipm.org/vim/en/1.16.html
 [serde]: https://serde.rs/
 

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -98,6 +98,15 @@
 //!  * `use_serde` -- Feature to enable support for serialization and deserialization of quantities
 //!    with the [Serde][serde] crate. Disabled by default.
 //!
+//!    [Serde][serde] support for the `big*` and `rational*` underlying storage types requires
+//!    manually enabling the `serde` feature for the `num-rational` and `num-bigint` crates. To do
+//!    so, you can add one or both of the following lines to your `Cargo.toml`:
+//!
+//!    ```toml
+//!    num-rational = { version = "*", features = ["serde"] }
+//!    num-bigint = { version = "*", features = ["serde"] }
+//!    ```
+//!
 //! [si]: https://jcgm.bipm.org/vim/en/1.16.html
 //! [serde]: https://serde.rs/
 //!

